exercises/supervisor_and_genserver_drills.livemd

```diff
@@ -133,43 +133,44 @@ It should:
 
 * Define a `State.get/1` function that uses [GenServer.call/3](https://hexdocs.pm/elixir/GenServer.html#call/3) and a [GenServer.handle_call/3](https://hexdocs.pm/elixir/GenServer.html#handle_call/3) callback function to retrieve state.
 * Define a `State.set/2` function that uses [GenServer.cast/2](https://hexdocs.pm/elixir/GenServer.html#cast/2) and a [GenServer.handle_cast/2](https://hexdocs.pm/elixir/GenServer.html#handle_cast/2) callback function to update state.
-* Define a `State.start_link/3` function with a [GenServer.init/1](https://hexdocs.pm/elixir/GenServer.html#init/1) callback function to initialize the [GenServer](https://hexdocs.pm/elixir/GenServer.html) with a configurable state.
+* Define a `State.start_link/3` function with a `State.init/1` callback function to initialize the [GenServer](https://hexdocs.pm/elixir/GenServer.html) with a configurable state.
```

Comment:
There is documentation for the callback function, but the current reading has the wrong link. Our autolinker assumes a specific format but the callback documentation breaks the format.


It should be:
https://hexdocs.pm/elixir/GenServer.html#c:init/1

Reply:
I've modified the code to match my comment above. Will merge this in now!